gearcmd/worker.go

```diff
@@ -102,8 +102,9 @@ func (conf TaskConfig) doProcess(job baseworker.Job) error {
 	var err error
 	if conf.ParseArgs {
 		args, err = argsparser.ParseArgs(string(job.Data()))
+
 		if err != nil {
-			return err
+			return fmt.Errorf("Failed to ParseArgs: %s", err.Error())
```

Comment:
nit: instead of basing this off the function name, can we base it off what it's doing? e.g. "failed to parse arguments"